apps/nextjs-website/react-components/components/HeroCounter/HeroCounter.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { Box, Typography, Link } from '@mui/material';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+import { isJSX } from '../../types/common/Common.types';
+import ContainerRC from '../common/ContainerRC';
+import { HeroCounterProps } from '@react-components/types/HeroCounter/HeroCounter.types';
+import { BackgroundColor, TextColor } from '../common/Common.helpers';
+import { useTheme } from '@mui/material/styles';
+
+const HeroCounter = (props: HeroCounterProps) => {
+  const {
+    background,
+    theme = 'dark',
+    counterNumber = 0,
```

Comment:
```suggestion
    counterNumber,
```
I wouldn't give the counter number a default value.

If someone is using this component they need to have a number to display.